<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SportiduinoPQ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Connec">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Com Port</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="choiseCom">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>15</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>17</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>18</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>21</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>22</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>23</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>25</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>26</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>27</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>28</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>29</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>30</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>31</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="Log">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>-3</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Card</string>
         </attribute>
         <widget class="QPushButton" name="InitCard">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Init Card</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="cardLine">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QCheckBox" name="AutoIncriment">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>AutoIncriment</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ReadCard">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Read Card</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Station</string>
         </attribute>
         <widget class="QPushButton" name="SetNum">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Set Number</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="StationLine">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="SetStart">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>60</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Set Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SetFinish">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>100</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Set Finish</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ClearSt">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>160</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Clear St</string>
          </property>
         </widget>
         <widget class="QPushButton" name="CheckSt">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>160</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Check St</string>
          </property>
         </widget>
         <widget class="QPushButton" name="LogCard">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>220</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Dump Card</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SetTime">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Set Time</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ReadLog">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>220</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Read Dump</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SleepCard">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>280</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Sleep Card</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Settings</string>
         </attribute>
         <widget class="QComboBox" name="WorkTime">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>10</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>6 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24 hour</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>all time</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>not work</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Work Time</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Start / Finish</string>
          </property>
         </widget>
         <widget class="QComboBox" name="StartFinish">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>50</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>on</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>off</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Check InitTime</string>
          </property>
         </widget>
         <widget class="QComboBox" name="CheckInitTime">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>90</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>on</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>130</y>
            <width>131</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Card Capicity</string>
          </property>
         </widget>
         <widget class="QComboBox" name="CardCap">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>130</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>auto</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>120</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>170</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>AutoDel Set</string>
          </property>
         </widget>
         <widget class="QComboBox" name="AutoDel">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>170</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>off</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>on</string>
           </property>
          </item>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>210</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>New Pass</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>250</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Old Pass</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="NewPass">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>210</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="OldPass">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>250</y>
            <width>101</width>
            <height>31</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="PassCard">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>290</y>
            <width>191</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Pass Card</string>
          </property>
         </widget>
         <widget class="QPushButton" name="SaveSet">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>330</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Save Set</string>
          </property>
         </widget>
         <widget class="QPushButton" name="LoadSet">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>330</y>
            <width>91</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Load Set</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionChose_COM_port">
   <property name="text">
    <string>Chose COM-port</string>
   </property>
  </action>
  <action name="actionSave_log">
   <property name="text">
    <string>Save log</string>
   </property>
  </action>
  <action name="actionOpen_help_txt">
   <property name="text">
    <string>Open help.txt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
